.ci.yaml

```diff
@@ -1498,6 +1498,30 @@ targets:
       benchmark: "true"
     scheduler: luci
 
+  - name: Linux_android color_filter_cache_perf__e2e_summary
+    bringup: true
+    recipe: devicelab/devicelab_drone
+    presubmit: false
+    timeout: 60
+    properties:
+      tags: >
+        ["devicelab","android","linux"]
+      task_name: color_filter_cache_perf__e2e_summary
+      benchmark: "true"
+    scheduler: luci
+
+  - name: Linux_android color_filter_cache_perf__timeline_summary
```

Comment:
Unless we have a reason to measure both, we are leaning towards e2e on newer benchmarks because it is a lower overhead mechanism. There are a number of benchmarks that we measure both on, but those date from when the E2E mechanism was new and we wanted to gauge the consistency of the two mechanisms.

I think we're just looking at tracking frame timings and cache usage here so E2E can provide all of that information.